### PR TITLE
Docs for the controls.

### PR DESCRIPTION
**Pull request by Jeevuz**

### CODE REVIEW

**Comment by Jeevuz** [comment]
I had to remove requestFocus too to be able to merge.

File: rxpm/src/main/kotlin/me/dmdev/rxpm/widget/CheckControl.kt [hunk 1]
```diff
@@ -9,14 +9,36 @@ import io.reactivex.disposables.Disposable
 import me.dmdev.rxpm.PresentationModel
 
 /**
+ * Helps to bind a group of properties of a checkable widget to a [presentation model][PresentationModel]
+ * and also breaks the loop of two-way data binding to make the work with the check easier.
+ *
+ * You can bind this to any [CompoundButton] subclass using the familiar `bindTo` method
+ * in the [PresentationModel].
```
Comment (dmdevgo) [inline rxpm/src/main/kotlin/me/dmdev/rxpm/widget/CheckControl.kt]: in the [AndroidPmView]

File: rxpm/src/main/kotlin/me/dmdev/rxpm/widget/ClickControl.kt [hunk 2]
```diff
@@ -9,14 +9,37 @@ import io.reactivex.disposables.Disposable
 import me.dmdev.rxpm.PresentationModel
 
 /**
+ * Helps to bind a group of properties of any clickable widget to a [presentation model][PresentationModel].
+ *
+ * You can bind this to any [View] subclass using the familiar `bindTo` method
+ * in the [PresentationModel].
```
Comment (dmdevgo) [inline rxpm/src/main/kotlin/me/dmdev/rxpm/widget/ClickControl.kt]: in the [AndroidPmView]

File: rxpm/src/main/kotlin/me/dmdev/rxpm/widget/InputControl.kt [hunk 3]
```diff
@@ -15,6 +15,17 @@ import io.reactivex.disposables.Disposable
 import me.dmdev.rxpm.PresentationModel
 
 /**
+ * Helps to bind a group of properties of an input field widget to a [presentation model][PresentationModel]
+ * and also breaks the loop of two-way data binding to make the work with the input easier.
+ *
+ * You can bind this to an [EditText] or an [TextInputLayout] using the familiar `bindTo` methods
+ * in the [PresentationModel].
```
Comment (dmdevgo) [inline rxpm/src/main/kotlin/me/dmdev/rxpm/widget/InputControl.kt]: in the [AndroidPmView]

**Comment by Jeevuz** [comment]
Fixed the docs.

**Pull request merged**